### ISSUE TITLE
Modifying automations with GUI broken.

### ISSUE THREAD

**Issue by henriec** [home-assistant/frontend#7279]
## The request
Since the update (sadly I'm not sure it was 115.2 or 115.4) it is not possible to edit automations using the GUI (web interface). I only create and edit automations using the GUI. Since very recently when I edit an automation (for example correct 1 character in the name/alias) all automations are gone except one. The modification is perfectly fine in the automations.yaml, but the GUI only shows (what appears to be) the most recently added one.  Only after a reboot everything shows up nicely. That's only until you touch one automation....

## The alternatives
- it's broken. Description tells everything.

## Additional information1
- not possible...
- btw, this is not a feature request, but a bug in my opinion... I don't see how to change the labels...

**Comment by zsarnett** [comment]
Please open a bug issue. When clicking new issue you are prompted with options
